pkg/sql/parser/sql.y

```diff
@@ -15810,6 +15883,7 @@ unreserved_keyword:
 | ENCODING
 | ENCRYPTED
 | ENCRYPTION_PASSPHRASE
+| ENCRYPTION_INFO_DIR
```

Comment:
Please add these to the `bare_label_keyword` list as well.

Reply:
Done.

Reply:
also, you mentioned in some meeting today that you'd file/ or work on a stability period issue that would create a linter for syntax changes right? I imagine we also need to add a bunch of backup/restore keywords to this list as well.